docs/Integration-with-CxSCA.md

```diff
@@ -171,4 +172,11 @@ In order to open SCA security tickets, set the bug tracker in cxflow app.yml fil
 * local zip scan:
   * -scan --app=MyApp --cx-team="my-team" --cx-project="project" --f="/Users/myProjects/project"
 * get latest scan results:
-  * --project --app=MyApp --cx-team="my-team" --cx-project="project" ([use 'project' command](https://github.com/checkmarx-ltd/cx-flow/blob/develop/src/main/java/com/checkmarx/flow/dto/ScanRequest.java))
\ No newline at end of file
+  * --project --app=MyApp --cx-team="my-team" --cx-project="project" ([use 'project' command](https://github.com/checkmarx-ltd/cx-flow/blob/develop/src/main/java/com/checkmarx/flow/dto/ScanRequest.java))
+
+
+## <a name="zipFolderScan">SCA ZIP folder scan</a>
+In order to change the default Cx-Flow SCA scan behaviour and to perform a SCA ZIP scan, the next configuration property should be added underneath the sca configuration section:
```

Comment:
The official product name is CxFlow (without a dash), so it's better to use the name without the dash throughout the docs for consistency.

Reply:
Right.